pantab/_types.py

```diff
@@ -62,3 +62,17 @@
 _pandas_types[
     _ColumnType(tab_api.SqlType.text(), tab_api.Nullability.NOT_NULLABLE)
 ] = "string"
+
+
+def _get_pandas_type(column_type: _ColumnType) -> str:
+    if column_type in _pandas_types:
+        return _pandas_types[column_type]
+
+    varchar = tab_api.SqlType.varchar(65535)
+    if column_type.type_.tag == varchar.tag:
```

Comment:
I think you can just simplify this as `if column_type.type_.tag == tab_api.TypeTag.VARCHAR`

https://tableau.github.io/hyper-db/lang_docs/py/tableauhyperapi.html#tableauhyperapi.TypeTag

Reply:
Simplified the check.